packages/hint-doctype/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "hint to validate if the doctype is correct",
+  "devDependencies": {
+    "@hint/utils-tests-helpers": "^1.0.0",
+    "@types/debug": "0.0.30",
+    "@types/node": "8.0.14",
+    "ava": "^0.25.0",
+    "cpx": "^1.5.0",
+    "eslint": "^5.5.0",
+    "eslint-plugin-markdown": "^1.0.0-beta.7",
+    "eslint-plugin-typescript": "^0.12.0",
+    "markdownlint-cli": "^0.13.0",
+    "npm-link-check": "^2.0.0",
+    "npm-run-all": "^4.1.2",
+    "nyc": "^13.0.1",
+    "rimraf": "^2.6.2",
+    "hint": "^3.3.2",
+    "typescript": "^3.0.3",
+    "typescript-eslint-parser": "^18.0.0"
+  },
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "dist/src"
+  ],
+  "homepage": "https://webhint.io/",
+  "keywords": [
+    "hint",
+    "hint",
+    "doctype",
+    "doctype-hint"
+  ],
+  "license": "Apache-2.0",
+  "main": "dist/src/hint.js",
+  "name": "@hint/hint-doctype",
+  "nyc": {
+    "extends": "../../.nycrc"
+  },
+  "peerDependencies": {
+   "hint": "^3.3.2"
+  },
+  "private": true,
+  "repository": "webhintio/hint",
+  "scripts": {
+    "build": "npm run clean && npm-run-all build:*",
+    "build-release": "npm run clean && npm run build:assets && tsc --inlineSourceMap false --removeComments true",
+    "build:assets": "cpx \"./{src,tests}/**/{!(*.ts),.!(ts)}\" dist",
+    "build:ts": "tsc",
```

Comment:
@CKGrafico can you double check these scripts with the one in any other hint?

at least `build:ts` has to be `tsc -b`.

This is something we need probably change in the templates.

Reply:
Ok thanks, I think that the template that I've used was not updated. Now I'm using `hint-minified-js` as the reference.